src/wasm/tf-builder.cc

```diff
@@ -497,6 +497,16 @@ TFNode* TFBuilder::Unop(WasmOpcode opcode, TFNode* input) {
     case kExprI32Clz:
       op = m->Word32Clz();
       break;
+    case kExprI32Popcnt:
+      {
+        compiler::OptionalOperator optionalOperator = m->Word32Popcnt();
```

Comment:
You can just do if (m->Word32Popcnt().IsSupported()) in the if condition. A redundant call won't be so bad if it makes the code look a little nicer.